src/scss/base/_base.scss

```diff
@@ -38,6 +38,22 @@ a.button {
     text-decoration: none;
 }
 
+.red {
```

Comment:
More helpful to give these semantic names such a "success" "error" "in progress" rather than naming them after colours.